src/CarbonAware/test/model/LocationTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using CarbonAware.Model;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarbonAware.Tests.model;
+
+public class LocationTests
+{
+    private static Location _defaultLocation = new Location() { Name = "eastus", Latitude = 34.123m, Longitude = 123.456m };
+
+    [Test]
```

Comment:
qq: Shouldn't be this simplified using only one method with `[TestCase(....)]` attribute?